tests/std/tests/P2674R1_is_implicit_lifetime/test.compile.pass.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <type_traits>
+
+enum UnscopedEnum { Enumerator };
+enum struct ScopedEnum { Enumerator };
+class Class {};
+class IncompleteClass;
+class UserProvidedDestructorClass
+{
+	~UserProvidedDestructorClass() {}
+};
+
+using namespace std;
+
+int main()
+{
+#ifdef __cpp_lib_is_implicit_lifetime 
+	// this is a test to test whether
+	// is_implicit_lifetime_v produces desired results
+	// compiles under std namespace
+
+	// Basics

```

Comment:
It's perhaps better to test `is_implicit_lifetime` (the class template) in some way.